pkg/controller/nodeipam/node_ipam_controller.go

```diff
@@ -110,7 +110,7 @@ func NewNodeIpamController(
 		glog.Fatal("Controller: Must specify --cluster-cidr if --allocate-node-cidrs is set")
 	}
 	mask := clusterCIDR.Mask
-	if maskSize, _ := mask.Size(); maskSize > nodeCIDRMaskSize {
+	if maskSize, _ := mask.Size(); allocatorType != ipam.CloudAllocatorType && maskSize > nodeCIDRMaskSize {
```

Comment:
It's clearer reading the code to do this:

```
if allocatorType != ipam.CloudAllocatorType {
  // Note: CloudAllocatorType does not rely on clusterCIDR or nodeCIDRMaskSize for allocation.
  if maskSize, _ := mask.Size(); maskSize > nodeCIDRMaskSize {
    ... 
  }
}

```

Reply:
Updated. Thanks

Reply:
I don't see your updates? Can you put the comment at least.

Reply:
My bad. Should be able to see the change now.